LabRecorder.cfg

```diff
@@ -12,13 +12,12 @@
 ; The BIDS syntax is: path/to/CurrentStudy/sub-%p/ses-%s/eeg/sub-%p_ses-%s_task-%b[_acq-%a]_run-%r_eeg.xdf
 ;
 ; If StorageLocation is not provided then the default is QStandardPaths::DocumentsLocation/CurrentStudy/exp%n/untitled.xdf
-
-;StorageLocation=C:/Recordings/CurrentStudy/exp%n/untitled.xdf
+; StorageLocation=C:/Recordings/CurrentStudy/sub-%p/ses-%s/behav/sub-%p_ses-%s_task-%b[_acq-%a]_run-%r_eeg.xdf
 
 ; === Study Root ===
 ; You can optionally provide only the study root folder and it will be used to build a file name automatically.
 ; The default filename builder is Legacy (see above).
-; StudyRoot=C:/Recordings/CurrentStudy
+StudyRoot=C:/Recordings/CurrentStudy
```

Comment:
This should be left as a disabled example because it's an invalid path on 2/3 platforms.